src/content/docs/security/index.mdx

```diff
@@ -103,8 +103,8 @@ that significantly reduces known vulnerabilities in the wild.
 ## Ecosystem
 
 The Tauri organization provides and maintains more than just the Tauri
-repository and to ensure that our efforts to provide a reasonable secure
-multi platform application framework we need to go some extra miles.
+repository, and to ensure we provide a reasonable secure
+multi platform application framework, we make sure to go some extra miles.
```

Comment:
i feel like "need" was a bit more correct here but i'm not the original author so idk